device-types/Nokia/7220-IXR-H2-128QSFP28.yaml

```diff
@@ -0,0 +1,302 @@
+---
+manufacturer: Nokia
+model: 7220 IXR-H2 128QSFP28
+slug: nokia-7220-ixr-h2-128qsfp28
+part_number: 3HE16421AA
+u_height: 4
+is_full_depth: true  # 21.65"
+weight: 65.78
```

Comment:
Do you have the `airflow` data point for this device or is it changeable based on the fan module?

Reply:
H2 supports both front-to-rear and rear-to-front FAN modules